kuka_rox_hardware_interface/src/robot_manager_node.cpp

```diff
@@ -46,64 +48,60 @@ RobotManagerNode::RobotManagerNode()
     "robot_manager/is_configured",
     is_configured_qos);
 
-  control_mode_map_.emplace(
-    std::make_pair(
-      ExternalControlMode::POSITION_CONTROL,
-      std::vector<std::string>(STANDARD_MODE_IF_SIZE)));
-  control_mode_map_.emplace(
-    std::make_pair(
-      ExternalControlMode::JOINT_IMPEDANCE_CONTROL,
-      std::vector<std::string>(IMPEDANCE_MODE_IF_SIZE)));
-  control_mode_map_.emplace(
-    std::make_pair(
-      ExternalControlMode::TORQUE_CONTROL,
-      std::vector<std::string>(STANDARD_MODE_IF_SIZE)));
-
-  // TODO(Svastits): change to dynamic parameter after control mode changes are supported
-  this->registerStaticParameter<int>(
+  control_mode_pub_ = this->create_publisher<std_msgs::msg::UInt32>(
+    "control_mode", rclcpp::SystemDefaultsQoS()
+  );
+
+  control_mode_change_finished_ = false;
```

Comment:
initialize this either in the constructor's initializer list or in the class definition

Reply:
done